src/TestStack.White/InputDevices/BareMetalKeyboard.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using TestStack.White.WindowsAPI;
+
+namespace TestStack.White.InputDevices
+{
+    //BUG: KeysConverter
+    /// <summary>
+    /// Represents Keyboard attachment to the machine.
+    /// </summary>
+    public static class BareMetalKeyboard
```

Comment:
Everything in this class is internal, make this internal and rename to `KeyboardWin32`?